sdk/translation/azure-ai-translation-document/tests/conftest.py

```diff
@@ -41,8 +43,8 @@ def add_sanitizers(test_proxy):
     # This is unnecessary for AzureCloud where each storage account is deleted at the end of testing
     if is_live() and os.getenv("TRANSLATION_ENVIRONMENT") == "Dogfood":
```

Comment:
Not blocking this PR, but judging by the comment above, it looks like we probably don't need this code anymore since we're no longer testing in Dogfood.